### ISSUE TITLE
NTR:Unilateral frontal polymicrogyria

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4563]
label:Unilateral frontal polymicrogyria (synonym: ) ID NA parent Frontal polymicrogyria def Unilateral polymicrogyria with anterior > posterior gradient extending from frontal poles posteriorly to precentral gyrus and inferiorly to frontal operculum [20301504] COST:neuromig,

**Comment by pnrobinson** [comment]
@Renske-Oegema Is it sufficient to use the modifier terms for this? https://hpo.jax.org/app/browse/term/HP:0012833 the HPO tries to avoid having separate terms for uni/bilateral unless there is an important distinction for differential diagnosis
